src/synapse/azext_synapse/_params.py

```diff
@@ -0,0 +1,162 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+from azure.cli.core.commands.parameters import name_type, tags_type, get_three_state_flag, get_enum_type, \
+    resource_group_name_type
+
+
+# pylint: disable=too-many-statements
+def load_arguments(self, _):
+    # synapse spark
+
+    for scope in ['batch', 'session', 'session-statement']:
+        with self.argument_context('synapse spark ' + scope) as c:
+            c.argument('workspace_name', help='The name of the workspace.')
+            c.argument('spark_pool_name', help='The name of the spark pool.')
+
+    for scope in ['synapse spark batch', 'synapse spark session']:
+        with self.argument_context(scope + ' create') as c:
+            c.argument('artifact_id', help='The artifact id.')

```

Comment:
Arguments that end in `-id` should be GUIDs. There are multiple parameters ending with `-id`, please confirm they use guid. Otherwise, you may want to change the option name.

Reply:
> Arguments that end in `-id` should be GUIDs. There are multiple parameters ending with `-id`, please confirm they use guid. Otherwise, you may want to change the option name.

fixed